groupMembershipControl.js

```diff
@@ -87,7 +87,8 @@ export async function renderGroupMemberships (person, context) {
     if (!book) {
       book = kb.any(undefined, ns.vcard('includesGroup'))
       if (!book) {
-        throw new Error('findBookFromGroups: Cant find address book which this group is part of')
+        // throw new Error('findBookFromGroups: Cant find address book which this group is part of')
```

Comment:
Wrapping things in single-quotes should not turn `can't` into `cant`, which is a distinct word with a different meaning. Either the enclosed apostrophe should be escaped, or it should be rephrased, as here --
```suggestion
        // throw new Error('findBookFromGroups: Cannot find address book which this group is part of')
```

(I know this is now commented out, but as long as it's retained in the code, it should be as correct as possible.)